Jellyfin.Plugin.Bookshelf/Providers/ComicBookInfo/ComicBookInfoProvider.cs

```diff
@@ -178,6 +169,41 @@ private MetadataResult<Book> SaveMetadata(ComicBookInfoFormat comic)
             }
         }
 
+        /// <inheritdoc />
+        public void ReadPeopleMetadata(ComicBookInfoMetadata comic, MetadataResult<Book> metadataResult)
+        {
+            foreach (var person in comic.Credits)
+            {
+                if (person.Person is null || person.Role is null)
+                {
+                    continue;
+                }
+
+                if (person.Person.Contains(',', StringComparison.InvariantCultureIgnoreCase))
+                {
+                    var name = person.Person.Split(',');
+                    person.Person = name[1].Trim(' ') + " " + name[0].Trim(' ');
+                }
+
+                var personInfo = new PersonInfo { Name = person.Person, Type = person.Role };
+                metadataResult.AddPerson(personInfo);
+            }
+        }
+
+        /// <inheritdoc />
+        public string? ReadCultureInfoInto(string language)
+        {
+            try
+            {
+                return new CultureInfo(language).DisplayName;
```

Comment:
Use the static method `CultureInfo.GetCultureInfo` instead. It caches internally.